src/libsyncengine/reconciliation/platform_inconsistency_checker/platforminconsistencycheckerutility.h

```diff
@@ -24,14 +24,17 @@
 
 namespace KDC {
 
+// TODO : this class should not be a singleton...
```

Comment:
Could elaborate just a little bit? You certainly have a compelling argument why it shouldn't be a singleton.

Reply:
Done